15-Sprint-Mission/index.html

```diff
@@ -0,0 +1,114 @@
+ 
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="home.css">
+</head>
+<body>
+    <header>
+
+    <a href="/">
+        <img src="images/logo/panda-market-logo.png" alt="판다마켓홈" width="153">
+    </a>
+      <a href="login.html" id="loginLinkButton" class="button">로그인</a>
+    </header>
+    <main>
+      <section id="hero" class="banner">
+        <div class="wrapper">
+            <h1>
+                "일상의 모든 물건을"
+                <br>
+                "거래해 보세요"
+            </h1>
+            <a href="items.html" class="button pill-button">구경하러 가기</a>
+        </div>
+      </section>
+      <section id="features" class="wrapper">
+        <div class="feature">
+     
+          <img src="images/home/feature1-image.png" alt="인기 상품" width="50%">
+          <div class="feature-content">
+            <h2 class="feature-tag">Hot item</h2>
+            <h1>
+                "인기 상품을"
+                <br>
+                "확인해 보세요"
+            </h1>
+            <p class="feature-description">
+                "가장 HOT한 중고거래 물품을"
+                <br>
+                "판다마켓에서 확인해 보세요"
+            </p>
+          </div>
+        </div>
+        <div class="feature">
+            <div class="feature-content">
+                <h2 class="feature-tag">Search</h2>
+                <h1>
+                    "구매를 원하는"
+                    <br>
+                    "상품을 검색하세요"
+                </h1>
+                <p class="feature-description">
+                    "구매하고 싶은 물품은 검색해서"
+                    <br>
+                    "쉽게 찾아보세요"
+                </p>
+            </div>
+            <img src="images/home/feature2-image.png" alt="검색 기능" width="50%">
+        </div>
+        <div class="feature">
+            <img src="images/home/feature3-image.png" alt="판매 상품 등록" width="50%">
+            <div class="feature-content">
+
+                <h2 class="feature-tag">Register</h2>
+                <h1>
+                    "판매를 원하는"
+                    <br>
+                    "상품을 등록하세요"
+                </h1>
+                <P class="feature-description">
+                    "어떤 물건이든 판매하고 싶은 상품을"
+                    <br>
+                    "쉽게 등록하세요"
+                </P>
+            </div>
+        </div>
+      </section>
+      <section id="bottomBanner" class="banner">
+        <div class="wrapper">
+            <h1>
+                "믿을 수 있는"
+                <br>
+                "판다마켓 중고거래"
+            </h1>
+        </div>
+      </section>
+    </main>
+    <footer>
+        <div>©codeit - 2024</div>
+        <div id="footerMenu">
+
+            <a href="privacy.html">Privacy Policy</a>
+            <a href="faq.html">FAQ</a>
+        </div>
+        <div id="socialMedia">
+
+        <a href="https://www.facebook.com/" target="_blank" rel="noopener noreferrer"><img src="images/social/facebook-logo.svg" alt="페이스북" width="20">
```

Comment:
❗️ 수정요청
여는 태그가 있으면 닫는 태그도 있어야합니다~

```suggestion
        <a href="https://www.facebook.com/" target="_blank" rel="noopener noreferrer"><img src="images/social/facebook-logo.svg" alt="페이스북" width="20" /></a>
```